packages/codemods/src/codeMods/__tests__/ComponentToCompat/ComponentToCompat.test.ts

```diff
@@ -0,0 +1,84 @@
+import { Project } from 'ts-morph';
+import * as Root from '../mock/compat/mockIndex';
+import {
+  IRawCompat,
+  runComponentToCompat,
+  getNamedExports,
+  buildCompatHash as buildHash,
+  repathNamedImports,
+  repathPathedImports,
+} from '../../mods/ComponentToCompat/CompatHelpers';
+import * as Button from 'office-ui-fabric-react/lib-commonjs/Button';
+
+const createComponentToCompat = (compat: IRawCompat) => {
+  return {
+    oldPath: './mockIndex',
+    newComponentPath: 'compat/Button',
+    namedExports: getNamedExports(compat.namedExports),
+  };
+};
+
+describe('Component to compat', () => {
+  let project: Project;
+  beforeEach(() => {
+    project = new Project();
+    project.addSourceFilesAtPaths(`${process.cwd()}/**/__tests__/mock/compat/*.tsx`);
+  });
+
+  it('correctly repaths from exact known import', () => {
+    const file = project.getSourceFileOrThrow('ImportsStuff.tsx');
+    const hash = buildHash([{ componentName: 'Button', namedExports: Root }], createComponentToCompat);
+    repathPathedImports(file, hash);
+    expect(file.getFullText()).toContain('compat/Button');
```

Comment:
this is not a robust enough check. meaning we could been doing wrong things can still have the test passing.

I think we should take a snapshot for `file.getFullText()` instead

Reply:
similar for other tests

Reply:
I'll iterate through the imports and check it directly instead of using a snapshot.

Reply:
i'd suggest at least have one test that takes a full snapshot of the file. perhaps a test for `runComponentToCompat`

Reply:
I'd prefer to not add snapshots just yet. It might be a good idea in the future but for now I the checks to be explicit.